src/target/codegen.cc

```diff
@@ -83,17 +85,19 @@ class ModuleSerializer {
 
     for (const auto& group : mod_group_vec_) {
       ICHECK_NE(group.size(), 0) << "Every allocated group must have at least one module";
-      if (!group[0]->IsDSOExportable()) {
```

Comment:
We need two modes here. If the 

- export mode (when we call export_library): Then we save as lib as long as it is DSOExportable.
- serialize mode (when we serialize): Always serialize when it is binary serializable. report an error when it is not, since we cannot recover the model